Directory.Build.props

```diff
@@ -5,7 +5,6 @@
 
   <Import Project="version.props" />
   <Import Project="build\dependencies.props" />
-  <Import Project="build\sources.props" />
```

Comment:
Curious? Why is nuget.config better?

Reply:
ok dope

Reply:
Friendship ended with `RestoreSources`.
Now `NuGet.config` is my best friend.

Reply:
I still like restore sources better. There are too many files.